inc/options/class-wpseo-option-wpseo.php

```diff
@@ -119,6 +124,70 @@ public static function get_instance() {
 		return self::$instance;
 	}
 
+	/**
+	 * Add filters to make sure that the option is merged with its defaults before being returned.
+	 *
+	 * @return void
+	 */
+	public function add_option_filters() {
+		parent::add_option_filters();
+
+		$hookname = "option_{$this->option_name}";
+		$callback = array( $this, 'verify_features_against_network' );
```

Comment:
You can make a method for this because you are using it 4 times. A method like `get_filter_callback` would be nice for this.